deploy/terraform/main.tf

```diff
@@ -61,7 +61,7 @@ module "resources" {
 
 terraform {
   backend "azurerm" {
-    resource_group_name  = "ifrcpgoterraform001rg"
+    resource_group_name  = "ifrctgo002rg"
```

Comment:
What was this change for @szabozoltan69? cc @batpad

Reply:
Has the storage account been migrated to the new resource group? Otherwise terraform may try to create new resources.

Reply:
Yes, storage account and also acr was migrated to that resource group (which is not new though).